vowpalwabbit/v_array.h

```diff
@@ -151,41 +172,69 @@ struct v_array
   inline size_t size() const { return _end - _begin; }
   inline size_t capacity() const { return end_array - _begin; }
 
-  void resize(size_t length)
+  // maintain the original (deprecated) interface for compatibility. To be removed in VW 10
+  //   VW_DEPRECATED(
```

Comment:
Resize is still used in lots of places and I haven't removed them yet. Since warnings are error this will break Windows CI. But it is commented out to show the intention.